src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@ public interface CinemaHallDao {
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    boolean update(CinemaHall cinemaHall);
```

Comment:
Consider changing the return type of the `update` method to `void` or `CinemaHall` to better reflect the operation's outcome. Returning a `boolean` might not be necessary if the method is expected to always succeed or throw an exception otherwise.